DEVELOPMENT.md

```diff
@@ -45,7 +45,7 @@ pod hostname (default.ds-idrepo-0.ds-idrepo.cluster.local, for example) In dev m
 kubectl port-forward ds-idrepo-0 1636
 ```
 
-This allows the opertor running on your desktop to communicate with the directory server. This is needed
+This allows the operator running on your desktop to communicate with the directory server. This is needed
 for any LDAP functionality.
 
```

Comment:
We need some docs somewhere to describe what this feature flag does. If we want to wait until its fully done that's cool - just want to call it out. 

I gather this will change the spec as well, so we need make sure we don't "release" this when creating a new version - it has to be pre-release

Reply:
See my comment to Jhonattan.  I'm wondering if I should leave out the option for the user to provide these values? It might be a bit messy as the advertised listen address would need the pod variable in the string to keep it unique.  If you think its worth having then I'll keep it in and document it in the readme

Reply:
I've removed the option for a user to specify this value.  Now it's just dynamically changes whether mcs is enabled or not.  This has simplified the condition too. No need for docs for this.